instrumentation/jetty-httpclient/jetty-httpclient-9.2/testing/build.gradle.kts

```diff
@@ -0,0 +1,19 @@
+plugins {
+  id("otel.java-conventions")
+}
+
+//Jetty client 9.2 is the best starting point, HttpClient.send() is stable there
+val jettyVers_base9 = "9.2.0.v20140526"
+
+dependencies {
+  api(project(":testing-common"))
```

Comment:
Removed exclusions that weren't needed (after removing jetty from testing-common)